src/site/content/en/blog/service-worker-mindset/index.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Service Worker Mindset
+subhead: How to think when thinking about Service Workers.
+authors:
+  - geddski
+date: 2019-05-28
+hero: hero.jpg
+hero_position: center
+alt: Service Workers lining up.
+description: |
+  Working with Service Workers is new and unfamiliar for many web devs. Here's some tips for wrapping your mind around them.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - service-worker
+  - PWA
+  - offline
+  - caching
+---
+
+Service Workers are powerful and absolutely worth mastering. They'll enable you to deliver an entirely new level of experience to your users. But Service Workers are unlike anything most of us web devs are used to. They come with a steep learning curve and a handful of snags you've got to watch out for.
```

Comment:
Suggest lowercasing service workers, except when starting a sentence.

I noticed MDN and our own articles refer to them as "service workers".
e.g. https://developers.google.com/web/fundamentals/primers/service-workers/

Reply:
but... but... my little fingers worked so hard to cap all those! ;)

Reply:
changed